src/ui/popup/manage-settings-page/export-settings.tsx

```diff
@@ -0,0 +1,21 @@
+import {m} from 'malevic';
+import {ViewProps} from '../types';
+import {Button} from '../../controls';
+
+export default function ExportButton(props: ViewProps) {
+    function exportSettings() {
+        const data = JSON.stringify(props.data.settings);
+        const a = document.createElement('a');
```

Comment:
Move file save function into `src/ui/utils.ts` use arguments `name: string` and `content: string`